Back port changes to 2.3.x (#830)
magento/merchdocs: src/stores/store-urls-custom-admin.md

1. After the changes are saved, **Sign Out** of the Admin. Then, log back in using the new Admin URL and path.

## Method 2: Change the Admin Path from the server command line

1. Open the `app/etc/env.php` file in a text editor, and change the value of the `frontName` parameter of the `backend` section. Then, save the file.

   Make sure to use only lowercase characters.

   {:.bs-callout-info}
   This method allows you to change the Admin Path, but not the Admin URL.

   - **Default Admin Path**
      ```php?start_inline=1
      'backend' => [
       'frontName' => 'admin'
      ],
      ```
      {: .no-copy}

   - **New Admin Path**
      ```php?start_inline=1
      'backend' => [
          'frontName' => 'backend'
      ],
      ```
      {: .no-copy}

1. Use one of the following methods to clear the Magento cache:

   - On the _Admin_ sidebar, go to **System** > _Tools_ > **Cache Management**. Then, click **Flush Magento Cache**.
   - On the server, execute the following:
      ```terminal
      php bin/magento cache:flush
      ```

   {:.bs-callout-info}

>    Note that the changes made using the Method 1 will have the priority over the changes made in the `app/etc/env.php` file.

Reviewer: ```suggestion
   Note that the changes made using Method 1 will have priority over the changes made in the `app/etc/env.php` file.
```

Author: good catch... will have to go back and fix that in master

## Restore the default Admin URL and Admin Path

> In case you have set an invalid Admin URL or an Admin Path and lost access to the backend, there is a way to fix this from the command line.

Reviewer: ```suggestion
If you have set an invalid Admin URL or Admin Path and lost access to the backend, you can fix this from the command line.
```
